scripts/utils.sh

```diff
@@ -115,9 +117,45 @@ function setNginxHostnames {
     export ODOO_HOSTNAME="localhost:8069"
     export SENAITE_HOSTNAME="localhost:8081"
     export SUPERSET_HOSTNAME="localhost:8088"
+    export ERP_NEXT_HOSTNAME="localhost:8082"
     echo "→ O3_HOSTNAME=$O3_HOSTNAME"
     echo "→ ODOO_HOSTNAME=$ODOO_HOSTNAME"
     echo "→ SENAITE_HOSTNAME=$SENAITE_HOSTNAME"
     echo "→ SUPERSET_HOSTNAME=$SUPERSET_HOSTNAME"
+    echo "→ ERP_NEXT_HOSTNAME=$ERP_NEXT_HOSTNAME"
 
 }
+
+function displayAccesURLsWithCredentials() {
```

Comment:
For the lack of a better idea of handling this. This wouldn't work on Pro because `ozone-urls-template.csv` isn't the same as FOSS one but some refactoring should make it work.

@rbuisson how should we handle this?

Reply:
This tooling will definitely hit some limits. Maybe the limit is now.
Does Docker Compose offer an API to know the exposed ports and services? (similar to what Gitpod does)

I would assume so. As the Docker orchestrator it probably knows that.
Otherwise we might want to look at registering the services to a service discovery service (this will also be needed as part of having various services exposing their available API endpoints (FHIR facade)

Let me take a look at that bash logic anyway and see if that can get us out of the woods for now.

Reply:
> Does Docker Compose offer an API to know the exposed ports and services? (similar to what Gitpod does)

Docker Compose itself does not provide an API to know the exposed ports and services. However, we can utilize Docker's API to get this information. [Docker provides a REST API that can be used to interact with the Docker Engine/daemon](https://docs.docker.com/engine/api/v1.44/), allowing us to inspect/list containers, networks, volumes, and services.